nengo/__init__.py

```diff
@@ -26,11 +26,13 @@
 from .learning_rules import PES, BCM, Oja, Voja
 from .params import Default
 from .probe import Probe
+from .processes import (BrownNoise, FilteredNoise, PresentInput, Process,
+                        WhiteNoise, WhiteSignal)
```

Comment:
This is a general question: when do we want things to become a "core" part of nengo, by which I mean be accessible simply with `nengo.<class>` rather than by `nengo.<module>.<class>`? Currently we don't have any of the distributions as core things; I think Gaussian and Uniform are so common that they could be core. With respect to processes, I would have `Process` and `WhiteSignal` for sure, `WhiteNoise` and `FilteredNoise` very probably, `PresentInput` probably, and `BrownNoise` maybe.

One thing to remember is once we promote something, it's hard to demote it without breaking code, whereas promoting things later is always fine because it will never break anything.